src/checkUsersTry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function checkUsersTry(usersTry, numberToGuess) {
```

Comment:
The name `checkUsersTry` and `usersTry` don't sound well. `checkUserAnswer` and `usersNumber` or `typedNumber` or anything similar.